app/src/main/java/com/example/dennis/leagueoflegions/GameView.java

```diff
@@ -104,7 +140,7 @@ private void draw()
                     canvas.drawText(DEBUG_TEXT[i], 10, 60+20*i, debugPaint);
                 }
             }
-
+            */
```

Comment:
Let's keep this uncommented so we can still use it later. To use, just write DEBUG_TEXT[0 to 4] = "some str" anywhere in this file.